### ISSUE TITLE
[Legitimate Site Blocked] openmeta.city

### ISSUE THREAD

**Issue by banadu3** [MetaMask/eth-phishing-detect#6221]
https://openmeta.city/

**Comment by banadu3** [comment]
Hi, Metamask

Please remove this domain from your warning list.

It's our company's newly launching service.

Thanks in advance.

-Jay

**Comment by devenblake** [comment]
Duplicate of #6219.

**Comment by banadu3** [comment]
Yes, it's duplicate of #6219.
I wish you could put it into Allowlist ASAP so that my company could launch this site quickly.
Unexpectedly being blocked by Metamask plugin causes the issue on our milestone of service.
Many thanks for your support.
-Jay.

**Comment by banadu3** [comment]
Hi, devenblake.

Can you let me know when your Allowlist patch is scheduled to merge with master branch, please?
It would be a big help for us to plan a roll-out date of the service.

Thanks,
Jay.
